tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -38,6 +39,29 @@ async def _run(self) -> StepStatus:
         return await super()._run(["metadata_service", "validate", "metadata.yaml"])
 
 
+class DeployOrchestratorStep(Step):
```

Comment:
```suggestion
class DeployOrchestrator(Step):
```
nit: I'm not sure we've been consistent in the step naming, but I find if redundant to add a Step suffix when the class explicitely inherits from Step

Reply:
Will change :)

However im going to put my reasoning on this naming out there just to context share for fun!

With my background being on the functional side, I find classes hard to understand at a glance. You never quite know whats going on until youve read the class, its subclasses, then trace the method you call. 

This is in contrast to functions where the types tell you what happened at a glance, this data type turned into this datatype can tell a pretty good story as you trace through code.

So I always find myself adding Parent Class names to child classes so I get the same glance when reading code.

"Oh this has step in the name, it probably subclasses Step"

Anyway, this is just a side effect of me coming from one style of programming into another.

Thus, TLDR, I will change the name here

Reply:
IMO the context in which these classes are instantiated helps you infer what's likely there parents. To be honest I don't have a strong opinion and if it makes you more comfortable with the code base feel free to keep it!